examples/hosts/app-integration/external-controller/src/view.ts

```diff
@@ -37,3 +38,36 @@ export function renderDiceRoller(diceRoller: IDiceRollerController, div: HTMLDiv
     // Use the diceRolled event to trigger the rerender whenever the value changes.
     diceRoller.on("diceRolled", updateDiceChar);
 }
+
+/**
+ * Render the user names of the members currently active in the session into the provided div
+ * @param audience - Object that provides the list of current members and listeners for when the list changes
+ * @param div - The div to render into
+ */
+export function renderAudience(audience: IAudience, div: HTMLDivElement) {
+    const wrapperDiv = document.createElement("div");
+    wrapperDiv.style.textAlign = "center";
+    wrapperDiv.style.margin = "70px";
+    div.appendChild(wrapperDiv);
+
+    const audienceDiv = document.createElement("div");
+    audienceDiv.style.fontSize = "20px";
+
+    const setAudienceMemberIds = () => {
```

Comment:
maybe updateAudienceMembers()? Or OnAudienceChange()?

Reply:
Updated to onAudienceChange 😄